conjure-go-client/httpclient/response_error_decoder_middleware.go

```diff
@@ -68,18 +68,3 @@ func (d restErrorDecoder) DecodeError(resp *http.Response) error {
 	// TODO(bmoylan) unmarshal conjure error
 	return werror.Error("server returned a status >= 400", werror.SafeParam("statusCode", resp.StatusCode))
 }
-
-// StatusCodeFromError retrieves the 'statusCode' parameter from the provided werror.
-// If the error is not a werror or does not have the statusCode param, ok is false.
-//
-// The default client error decoder sets the statusCode parameter on its returned errors. Note that, if a custom error
-// decoder is used, this function will only return a status code for the error if the custom decoder sets a 'statusCode'
-// parameter on the error.
-func StatusCodeFromError(err error) (statusCode int, ok bool) {
```

Comment:
So this was actually public api and I know for a fact we are using it in some places

Reply:
hrm ok -- needed it internally and can't have cycles, will move elsewhere

Reply:
You may need to decomp out the internal function and call that from this, as moving a function is also an API break.

Reply:
womp, yep